README.md

```diff
@@ -18,6 +18,25 @@ Those packages are built for:
 
 ## Installation
 
+### Repository key
+
+Packages provided by the repository are signed. In order to verify the integrity of the packages, you need to import the public key. First, verify that the fingerprint of `key.asc` matches `4EFC 5906 96CB 15B8 7C73  A3AD 82CC 8797 C838 DCFD`:
+
+```sh
+curl -fsSL https://pkgs.zabbly.com/key.asc | gpg --show-keys --fingerprint
+
+pub   rsa3072 2023-08-23 [SC] [expires: 2025-08-22]
+      4EFC 5906 96CB 15B8 7C73  A3AD 82CC 8797 C838 DCFD
+uid                      Zabbly Kernel Builds <info@zabbly.com>
+sub   rsa3072 2023-08-23 [E] [expires: 2025-08-22]
+```
+
+If so, dearmor the key and save it locally:
+
+```sh
+curl -fsSL https://pkgs.zabbly.com/key.asc | sudo gpg --dearmor -o /etc/apt/keyrings/zabbly.gpg
```

Comment:
Why dearmor though? apt handles armored keys just fine and they're much easier to handle in configuration management tools and backups when they're not binary blobs.

Reply:
I don't have a strong opinion on which should be used, I was under the (mistaken) assumption that binary format was the norm but it looks like it's an Ubuntu thing. Feel free to change it to whatever you prefer.

Reply:
Yeah, I'd prefer to keep the unarmored, I'm not a fan of having binary files in /etc :)